sdk/ml/azure-ai-ml/azure/ai/ml/_schema/pipeline/pipeline_datatransfer_job.py

```diff
@@ -36,7 +36,7 @@ def make(self, data: Any, **kwargs: Any):
 class PipelineDataTransferImportJobSchema(DataTransferImportJobSchema):
     outputs = fields.Dict(
         keys=fields.Str(),
-        values=UnionField([NestedField(OutputSchema)], allow_none=True),
+        values=UnionField([NestedField(OutputSchema), fields.Str()], allow_none=True),
```

Comment:
why any strings are accepted here?

Reply:
Just copy from other pipeline***job schema, will update to OutputBindingStr